entgql/annotation.go

```diff
@@ -46,8 +46,17 @@ func OrderField(name string) Annotation {
 }
 
 // Bind returns a binding annotation.
+//
+// No-op function to avoid breaking the existing schema.
+// You can safely remove this function from your scheme.
+// To disable Bind, use BindDisabled()
 func Bind() Annotation {
-	return Annotation{Bind: true}
+	return Annotation{}
+}
+
+// BindDisabled returns a disabled binding annotation.
+func BindDisabled() Annotation {
+	return Annotation{BindDisabled: true}
 }
 
```

Comment:
What do you think about accepting an optional parameter as follows:

```go
edge.To(...).
    Annotation(
       entgql.Bind() // Default to true. 
    )

edge.To(...).
    Annotation(
       entgql.Bind(true) // Same as Bind().
    )

edge.To(...).
    Annotation(
       entgql.Bind(false) // Allow overriding the default behaviour.
    )
```

Then, you can keep the `Bind` field but as a pointer, or use BindDisabled.

Reply:
Is Go has optional argument? Or you trying to said:

funct Bind(bind ...bool) {}

Reply:
exactly.

Reply:
> func Bind(bind ...bool) {}

I think it's a bit complicated to implement, and it's also kind of weird to use only the first element.

Reply:
I'm updated the PR with this commit https://github.com/ent/contrib/pull/217/commits/3c7a9d28cc1b43f48d94ed293fb9a02f0b970ee5
But It can be lead to the invalid case when User using it as `Annotation(entgql.MapTo("field1", "field2"), entgql.Bind(true))`